### PR TITLE
Fix flutter backend match check

### PR DESCRIPTION
**Pull request by d-uzlov**
Closes https://github.com/mlcommons/mobile_app_open/issues/142

### CODE REVIEW

File: flutter/cpp/flutter/dart_backend_match.cc [hunk 1]
```diff
@@ -8,9 +8,10 @@
 
 extern "C" struct dart_ffi_backend_match_result *dart_ffi_backend_match(
     const char *lib_path, const char *manufacturer, const char *model) {
+  LOG(INFO) << "checking backend '" << lib_path << "'...\n";
   ::mlperf::mobile::BackendFunctions backend(lib_path);
   if (*lib_path != '\0' && !backend.isLoaded()) {
-    std::cerr << "backend '" << lib_path << "' can't be loaded\n";
+    LOG(INFO) << "backend '" << lib_path << "' can't be loaded\n";
```
Comment (anhappdev) [inline flutter/cpp/flutter/dart_backend_match.cc:14]: The new line character here (and in other lines where it is appended to `LOG`) creates an empty line.

File: flutter/cpp/flutter/dart_backend_match.cc [hunk 2]
```diff
@@ -8,9 +8,10 @@
 
 extern "C" struct dart_ffi_backend_match_result *dart_ffi_backend_match(
     const char *lib_path, const char *manufacturer, const char *model) {
+  LOG(INFO) << "checking backend '" << lib_path << "'...\n";
```
Comment (anhappdev) [inline flutter/cpp/flutter/dart_backend_match.cc:11]: On iOS, there is only TFLite as default backend and the log shows
`I flutter/cpp/flutter/dart_backend_match.cc:11] checking backend ''...`
`I flutter/cpp/flutter/dart_backend_match.cc:67] backend '' matches`
Should we check for this case and make the log more understandable?

File: flutter/cpp/flutter/dart_backend_match.cc (hunk 1 above)
Comment (d-uzlov) [reply flutter/cpp/flutter/dart_backend_match.cc]: Fixed

File: flutter/cpp/flutter/dart_backend_match.cc (hunk 2 above)
Comment (d-uzlov) [reply flutter/cpp/flutter/dart_backend_match.cc]: Fixed

**Pull request merged**